src/Graveyard.tsx

```diff
@@ -36,7 +40,7 @@ export function Graveyard(props: GraveyardProps) {
         const sorted = burntSlugs.sort((a, b) => a.rank - b.rank);
 
         if (professionalGraveyard) {
-            return <ProfessionalGraveyard burntSlugs={sorted}/>;
+            return <ProfessionalGraveyard burntSlugs={sorted} traitNameMap={traitNameMap} attributes={attributes}/>;
```

Comment:
When we have more than one property being passed lets split this onto multiple lines

```suggestion
            return (
                <ProfessionalGraveyard
                    burntSlugs={sorted}
                    traitNameMap={traitNameMap}
                    attributes={attributes}
                />
            );
```

The braces aren't needed but looks better imo

Reply:
this isn't fixed